### PR TITLE
WIP fix: does not print fully-qualified of "java.lang"

### PR DESCRIPTION
**Pull request by monperrus**

### CODE REVIEW

File: src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java [hunk 1]
```diff
@@ -1731,7 +1731,10 @@ public void visitCtWildcardReference(CtWildcardReference wildcardReference) {
 	}
 
 	private boolean printQualified(CtTypeReference<?> ref) {
-		return ignoreImplicit || !ref.isImplicitParent();
+		// in Java, package "java.lang" has a special status
+		// it's the only one that does not need to fully-qualifed
```
Comment (seintur) [inline src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java]: qualified

Comment (monperrus) [reply src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java]: thanks

**Comment by monperrus** [comment]
123 failures:

* I don't want to change 123 assertions
* If we do so, we will also break client code

So I will simply try to stick to the previous behavior.

**Comment by monperrus** [comment]
not the good solution

**Pull request closed without merging**